packages/cursorless-engine/src/processTargets/modifiers/GlyphStage.ts

```diff
@@ -0,0 +1,53 @@
+import { GlyphModifier, Range } from "@cursorless/common";
+import { Target } from "../../typings/target.types";
+import { ModifierStageFactory } from "../ModifierStageFactory";
+import { ModifierStage } from "../PipelineStages.types";
+import { PlainTarget } from "../targets";
+
+export class GlyphStage implements ModifierStage {
```

Comment:
wait...why is this a modifier? I can't imagine why it wouldn't be a scope 🤔. Then you could say "every dollar", "next dollar", "first dollar", etc, all of which are very useful